### PR TITLE
Fix whitespace

### PR DESCRIPTION
**Pull request by alextd**
Removed trailing whitespace, fixed tabs where there should be spaces, spaces where there should be tabs. Space+tab where there should be just Tab.

Don't worry, I'm not just obsessing for no reason. I went to search for 'has "Nifel"', didn't find it, thought there was no mission line there. There actually is a mission with 'has(TAB)"Nifel". So I went and applied some regex to the data files to find extraneous or misplaced whitespace. Why doesn't this commit fix that line though? Well because it was already fixed and JUST NOW merged into master. Oh well, take my whitespace fixes.

An example of the egregious problems this fixes:

image

(And on a side note I'm searching for text editor that searches for any whitespace when you search for a space.)

### CODE REVIEW

**Comment by alextd** [comment]
Spellcheck test fail huh? Well I only changed the whitespace of that line, how did it pass spellcheck before? ... Or did it?

**Comment by Amazinite** [comment]
The `.codespell.exclude` file find exact matches. By changing the whitespace, you changed a line somewhere that was previously being excluded but no longer matches.

File: data/human/deep missions.txt [hunk 1]
```diff
@@ -2780,7 +2780,7 @@ mission "Home for Skadenga 1"
 			apply
 				payment -2000
 			`	The woman's face lights up, and you get the sense that she isn't used to hearing "yes" very often. "Oh, that is wonderful ser, just wonderful!"`
-			` 	She puts her finger to her ear and says, "Hjlod, we have a customer! Get the crawler warmed up!" The woman then drops her hand and waves to you. "Right this way, ser! And might I add - you will not be disappointed." She chatters excitedly about the cultural, artistic and scientific merits of the trip as she leads you toward a large pressure hatch.`
+			`	She puts her finger to her ear and says, "Hjlod, we have a customer! Get the crawler warmed up!" The woman then drops her hand and waves to you. "Right this way, ser! And might I add - you will not be disappointed." She chatters excitedly about the cultural, artistic and scientific merits of the trip as she leads you toward a large pressure hatch.`
```
Comment (MCOfficer) [inline data/human/deep missions.txt:2783]: You need to make the same change in `.codespell.exclude` L24

**Comment by Ferociousfeind** [comment]
The trailing triple-tabs tremendously... irk me. I can't believe those got in. That's three whole tabs, like four times.

**Review by Zitchas — APPROVED** [review]
The content-dude changes look good, although I can't speak for the code-side changes.

**Pull request merged**